src/rest.js

```diff
@@ -113,7 +113,7 @@ function del(config, auth, className, objectId) {
       if (hasTriggers || hasLiveQuery || className == '_Session') {
         return new RestQuery(config, auth, className, { objectId })
           .forWrite()
-          .execute()
+          .execute({ op: 'delete' })
```

Comment:
can you help me understand what is going on here?

Reply:
This is to indicate we’re doing a read for a delete. Otherwise we would not be able to fetch the object from the DB while the user is properly allowed to delete